app/Http/Controllers/MonicaController.php

```diff
@@ -4,21 +4,36 @@
 
 use App\Helpers\Products;
 use App\Models\Plan;
+use App\Services\UpdateSubscription;
+use Illuminate\Http\RedirectResponse;
 use Illuminate\Http\Request;
 use Inertia\Inertia;
+use Inertia\Response;
 
 class MonicaController extends Controller
 {
     public const PRODUCT = 'Monica';
 
-    public function index(Request $request)
+    /**
+     * Display Monica licences.
+     *
+     * @param  \Illuminate\Http\Request  $request
```

Comment:
You know that since the namespace is already imported, you could simply write Request $request without the full path?

Reply:
I've just followed Laravel documentation convention